src/js/utils.js

```diff
@@ -3,6 +3,10 @@ export const setCookie = (value) => {
   d.setTime(d.getTime() + 24 * 60 * 60 * 1000);
   const expires = 'expires=' + d.toUTCString();
   document.cookie = '_cookies_accepted=' + value + '; ' + expires;
+  if ((value == 'all' || value == 'performance') && typeof dataLayer == 'function') {
+    window.dataLayer = window.dataLayer || [];
```

Comment:
I guess I can remove this? as it will never be called?

Reply:
Absolutely, once its available we should push a pageview event

Reply:
Would you mind moving this logic to its own function please